src/tests/NXTTest.cpp

```diff
@@ -193,7 +195,8 @@ void NXTTest::AddTextureExpectation(const char* file, int line, const nxt::Textu
     deferred.rowPitch = rowPitch;
     deferred.expectation = expectation;
 
-    deferredExpectations.push_back(deferred);
+    deferredExpectations.push_back(std::move(deferred));
```

Comment:
They do now, that was when there was an emplace_back on one side and a push_back on the other.